### PR TITLE
Add long press and release event support

### PR DESCRIPTION
**Pull request by bolkedebruin**
This adds long press support and basic for release event support. This is done by overriding @final
methods of homeassistant's button for which it is
unclear why they are marked as such.

Attributes for type of click are now set to "long" for a long press and "single" for a short press.

### CODE REVIEW

File: custom_components/button_plus/button.py [hunk 1]
```diff
@@ -107,3 +132,24 @@ def device_info(self) -> DeviceInfo:
...
+
+    async def _async_long_press_action(self) -> None:
+        self._attr_click_type = "long"
+        await super()._async_press_action()
+
+    async def _async_press_action(self) -> None:
+        self._attr_click_type = "single"
+        await super()._async_press_action()
+
+    async def _async_release_action(self) -> None:
```
Comment (P-Storm) [inline custom_components/button_plus/button.py:144]: Why do we have this defined and passing the body. Is the abstract function throwing errors?

**Comment by P-Storm** [comment]
Otherwise, looking good. Also nice that you grouped the topics to one space :+1:

File: custom_components/button_plus/button.py (hunk 1 above)
Comment (bolkedebruin) [reply custom_components/button_plus/button.py:144]: Haha no, it's just that the functionality to do something with "release" is not fully implemented yet. It's the foundational layer.

**Review by P-Storm — APPROVED** [review]

**Review by koenhendriks — CHANGES_REQUESTED** [review]
Could you take a look at the CI errors?

```
Error: R] [SERVICES] Service long_press has no name and is not in the translations file
Error: R] [SERVICES] Service release has no name and is not in the translations file
Error: R] [TRANSLATIONS] Invalid strings.json: extra keys not allowed @ data['config']['services']. Got {'long_press': {'name': 'Long press', 'description': 'Long press on a button on button+'}, 'release': {'name': 'Release', 'description': 'Release of a button on button+'}}
required key not provided @ data['config']['step']. Got None
```

**Comment by bolkedebruin** [comment]
Done

**Comment by bolkedebruin** [comment]
ping @koenhendriks :-)

**Review by koenhendriks — APPROVED** [review]
Haven't been able to setup my development environment yet since I've moved, sorry for the delay. I'm reviewing this without feature tests now.

**Pull request merged**